internal/profiling/profiling.go

```diff
@@ -0,0 +1,28 @@
+package profiling
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/metal-toolbox/bioscfg/internal/log"
+)
+
+const (
+	Endpoint = "localhost:9091"
+)
+
+// Enable the profiling endpoint
+func Enable() {
+	go func() {
+		server := &http.Server{
+			Addr:              Endpoint,
+			ReadHeaderTimeout: 2 * time.Second, // nolint:gomnd // time duration value is clear as is.
```

Comment:
We should disable `gomnd`. It's of limited utility.

Reply:
This was copy/paste. I don't have that linter running anyway, so I removed it.